examples/create-react-app/package.json

```
@@ -9,8 +9,9 @@
     "eject": "react-scripts eject"
   },
   "dependencies": {
-    "@material-ui/core": "latest",
-    "clsx": "latest",
+    "@emotion/core": "^10.0.35",
+    "@emotion/styled": "^10.0.27",
```

So we don't need to keep it update to date and we get feedback when a new major is released?

```suggestion
    "@emotion/core": "latest",
    "@emotion/styled": "latest",
```